### PR TITLE
Feature/linkedin api updates 2023

### PR DESCRIPTION
**Pull request by fivetran-sheringuyen**
❗ ❗ 
We are currently waiting for data to verify the updates before merging and releasing the changes in this PR. If you are currently using our Linkedin Package and would like to test out this version of the Package, we would love to hear about your experience. Please feel free to comment below and let us know what your experience is! 

For those interested, you can test out this version by adding the below in your packages.yml and temporarily comment out your current version of `dbt_linkedin`.

```
packages:
  - git: https://github.com/fivetran/dbt_linkedin.git
    revision: feature/linkedin-api-updates-2023
    warn-unpinned: false
```
❗ ❗ 

**Are you a current Fivetran customer?** 

Fivetran employee 💃 

**What change(s) does this PR introduce?** 

## 🚨 Breaking Changes 🚨
Due to Linkedin Ads API [change in January 2023](https://learn.microsoft.com/en-us/linkedin/marketing/integrations/recent-changes?view=li-lms-2022-12#january-2023), there have been updates in the Linkedin Ads Fivetran Connector and therefore, updates to this Linkedin package. 

The following fields have been completely deprecated in the `stg_linkedin_ads__creative_history` model:
- `type`
- `call_to_action_label_type`
- `version_tag`

## Updates
- `linkedin_ads__creative_report` now includes `campaign.type as campaign_type` as a proxy for the previously used `creative.type` field.
- `linkedin_ads__creative_report` no longer includes the fields `creative.version_tag`, `creative.type` and `creative.call_to_action_label_type`.
- `linkedin_ads__creative_report` now leverages `report.creative_id` instead of `creative.creative_id`. 
- `linkedin_ads__campaign_report` now leverages `report.campaign_id` instead of `campaign.campaign_id`.
- `linkedin_ads__url_report` no longer includes the field `creative.version_tag` and now leverages `report.creative_id` instead of `creative.creative_id`.

## Under the hood
- `integration_tests/seed/linkedin_creative_history_data` has been updated to reflect new fields and deprecated fields updates
- Legacy fields have been updated respectively in the connector and `dbt_linkedin_source v0.7.0` includes modifications that could affect the below fields within the `linkedin_ads__creative_history` model:
  - `last_modified_at`
  - `created_at`
  - `status`
For more information, please refer to [dbt_linkedin_source PR #48](https://github.com/fivetran/dbt_linkedin_source/pull/48).

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [x] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [x] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Comment by fivetran-sheringuyen** [comment]
To-do:
- [ ] Docs to be generated before merge & release!

File: packages.yml
Comment (fivetran-joemarkiewicz) [inline packages.yml:3]: We should also update the README to reference this new version range of the source package.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-sheringuyen The changes look good; however, I will hold off on approving until we get confirmation from the product team that these are good to roll out.

Additionally, I had a few super minor comments to address in the meantime.

File: integration_tests/dbt_project.yml
Comment (fivetran-sheringuyen) [reply integration_tests/dbt_project.yml:2]: Good catch! Updated both parts of the README!

**Review by Csgoodman — APPROVED** [review]
Looks good to me! Very thorough change log :)

File: packages.yml
Comment (fivetran-joemarkiewicz) [reply packages.yml:3]: Reminder to update before merge

**Review by fivetran-joemarkiewicz — APPROVED** [review]
These changes look good to me! Thanks again @fivetran-sheringuyen

**Pull request merged**
